File: HorseRacing.Web/src/interfaces/api/contracts/model/game/requests/place-bet/place-bet-request-dto.ts
import type { BaseDto } from "~/interfaces/api/contracts/base/dto/base-dto";

export interface PlaceBetRequestDto extends BaseDto {
    /** Идентификатор игры */
    GameId: string;
    /** Идентификатор пользователя */
    UserId: string;
    /** Ставка пользователя */
    BetAmount: number;
    /** Масть лошади, которую выбрал пользователь */
    BetSuit: number;
}

File: HorseRacing.Contracts/Models/Game/Requests/PlaceBet/PlaceBetRequestDto.cs
using HorseRacing.Contracts.Base.Dto;

namespace HorseRacing.Contracts.Models.Game.Requests.PlaceBet
{
    public class PlaceBetRequestDto : BaseDto
    {
        /// <summary>
        /// Код пользователя
        /// </summary>
        public Guid UserId { get; set; }
        /// <summary>
        /// Код игры
        /// </summary>
        public Guid GameId { get; set; }
        /// <summary>
        /// Ставка
        /// </summary>
        public int BetAmount { get; set; }
        /// <summary>
        /// Масть
        /// </summary>
        public int BetSuit { get; set; }
    }
}
